src/Engine/GameEvents/CommanderAbilityEvent.java

```diff
@@ -45,9 +40,10 @@ public void performEvent(MapMaster gameMap)
   }
 
   /** Called by AbilityAction before `getEventAnimation`, `performEvent`, or `sendToListener`. */
-  public void generateAbilityEvents(MapMaster gameMap)
+  public GameEventQueue generateAbilityEvents(MapMaster gameMap)
   {
     gameEvents = myAbility.getEvents(gameMap);
+    return gameEvents;
   }
```

Comment:
Why not just throw out this function in favor of calling `myAbility.getEvents()` directly in `AbilityAction`?